tensorflow/g3doc/get_started/os_setup.md

```diff
@@ -370,7 +370,9 @@ After Docker is installed, launch a Docker container with the TensorFlow binary
 image as follows.
 
 ```bash
-$ docker run -it -p 8888:8888 gcr.io/tensorflow/tensorflow
+$ docker run -d -p 8888:8888 --name="tensor" gcr.io/tensorflow/tensorflow
+
+$ docker exec -it tensor bash
```

Comment:
@craigcitro - You have raised good points. 

As per the documentation (https://www.tensorflow.org/versions/r0.9/get_started/os_setup.html#docker-installation), it is assumed that the developer would use tensorflow docker image to isolate their host environment from the tensorflow packages. 

It is better to scope the two docker invocations separately. 

1. If you would wish to run the notebook as a server, we use the ```-it```. This will avoid separate invocation of ```docker logs``` on the existing container. 

2. The need for CLI is ideally to isolate the host (local) environment from the tensorflow packages. The user may want to use the python installation, and the installed packages in the container to test out code. In that case, ```--entrypoint /bin/bash``` is more cleaner. 

@vrv  - what do you think ?